### DISCUSSION TITLE
What do travelers want to see the most?

### DISCUSSION THREAD

**Discussion by rachelpr** [rachelpr/LeafSmart#100] [Ideas]
Let's talk about what travels might want to see the most, and how we want the integrated in our app?

**Comment by b1u3too** [comment]
First blusgh ideas from what's available through the Teleport API: [link](https://docs.google.com/document/d/1AByipwwDinR9Rxb6ogw78ijeqmQSRxuDIUQnw_suWyY/edit?usp=sharing)

**Reply by b1u3too** [reply]
Don't know what a blusgh is, but it stands 😊

**Comment by b1u3too** [comment]
AirBnB/hotel accommodations come to mind!
